libstuff/libstuff.cpp

```diff
@@ -2614,15 +2614,14 @@ int SQuery(sqlite3* db, const char* e, const string& sql, SQResult& result, int6
     }
     uint64_t elapsed = STimeNow() - startTime;
 
-    // Warn if it took longer than the specified threshold
+    // This code removing authTokens is a quick fix and should be removed once https://github.com/Expensify/Expensify/issues/144185 is done.
     string sqlToLog = sql;
-    if ((int64_t)elapsed > warnThreshold) {
-        // We should always avoid logging authTokens because they give access to accounts
-        pcrecpp::RE("\"authToken\":\"[0-9A-F]{400,1024}\"").GlobalReplace("\"authToken\":<REDACTED>", &sqlToLog);
-
-        // We remove anything inside "html" because we intentionally don't log chats
-        pcrecpp::RE("\"html\":\".*\"").GlobalReplace("\"html\":\"<REDACTED>\"", &sqlToLog);
+    // We should always avoid logging authTokens because they give access to accounts
+    pcrecpp::RE("\"authToken\":\"[0-9A-F]{400,1024}\"").GlobalReplace("\"authToken\":<REDACTED>", &sqlToLog);
```

Comment:
I am slightly concerned about running this code always now. Can we at least run it only if we're going to log the query?